gradle.properties

```diff
@@ -1,4 +1,4 @@
-ideaVersion = IU-2018.2
+ideaVersion=IU-2019.1.4
```

Comment:
Why 2019.1.4 and not 2019.1 ? Because if our minimum is 2019.1.4 then we need to update plugin.xml